nbfetch/handlers.py

```diff
@@ -2,29 +2,49 @@
 from tornado.escape import url_escape, url_unescape
 import traceback
 from urllib.parse import urljoin
-from notebook.base.handlers import IPythonHandler
+
 import threading
 import json
 import os
+from pathlib import Path
+import io
```

Comment:
Is this library being used?

Reply:
`io` is not. I can remove it. As a word of caution, I applied `black` code formatting to `handlers.py` so a lot of "changes" are going to show up that I did not make. That's my bad and I should have avoid doing so, to make the review more clear. 

Other than adding the `handlers::ExtensionHandler` mixin, the only real changes I made to `handlers.py` is the `HSHandler` class for review sake. All the other stuff should mainly be formatting changes.